scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -131,6 +131,9 @@ proposals:
         host: vcs.qa.suse.de
         user: "@@vcenter_user@@"
         password: "@@vcenter_password@@"
+        cluster_name: []
+        insecure: false
```

Comment:
we might want to set this to `true` otherwise it fails due to empty ca_cert field. I think we have no certificates yet in place.